src/google-oauth2.d.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Definition file for the node.js module
+ *
+ */
+interface GoogleOAuth2Options{
```

Comment:
Not entire sure how `ts` files work but it could look nicer if there is a space in front of the opening `{`.